ckks/bootstrapping/utils.go

```diff
@@ -0,0 +1,7 @@
+package bootstrapping
```

Comment:
Move to `utils` so we can use it elsewhere in the future ?

Reply:
done